File: sdks/java/selenium/src/test/java/support/tests/UFG.java
package support.tests;

import com.applitools.eyes.RectangleSize;
import com.applitools.eyes.Region;
import com.applitools.eyes.selenium.StitchMode;
import com.applitools.eyes.selenium.fluent.Target;
import org.openqa.selenium.By;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.BeforeTest;
import org.testng.annotations.Test;
import support.BaseSetup;

public class UFG extends BaseSetup {

    @BeforeClass
    public void classSetup() {
        UFG = true;
    }

    @Test
    public void window(){
        driver.get("https://applitools.github.io/demo/TestPages/FramesTestPage/");
        eyes.open(driver, "Applitools Support Matrix", "Window - UFG", new RectangleSize(700, 460));
        eyes.check(Target.window());
        eyes.close(true);
    }

    @Test
    public void region() {
        driver.get("https://applitools.github.io/demo/TestPages/FramesTestPage/");
        eyes.open(driver, "Applitools Support Matrix", "Region - UFG", new RectangleSize(700, 460));
        eyes.check(Target.region(new Region(50, 70, 90, 110)));
        eyes.close(true);
    }
}

File: sdks/python/selenium/tests/support/UFG.py
import pytest
from applitools.selenium import (Region, Target, VisualGridRunner)


@pytest.fixture(scope="function")
def eyes_runner_class():
    return VisualGridRunner(10)


def window(driver, eyes):
    driver.get("https://applitools.github.io/demo/TestPages/FramesTestPage/")
    conf = eyes.get_configuration()
    conf.test_name = "Window - UFG"
    conf.app_name = "Applitools Support Matrix"
    conf.viewport_size = {"width": 700, "height": 460}
    eyes.set_configuration(conf)
    eyes.open(driver)
    eyes.check(Target.window())
    eyes.close(raise_ex=True)


def region(driver, eyes):
    driver.get("https://applitools.github.io/demo/TestPages/FramesTestPage/")
    conf = eyes.get_configuration()
    conf.test_name = "Region - UFG"
    conf.app_name = "Applitools Support Matrix"
    conf.viewport_size = {"width": 700, "height": 460}
    eyes.set_configuration(conf)
    eyes.open(driver)
    eyes.check(Target.window().region(Region(50, 70, 90, 110)))
    eyes.close(raise_ex=True)